src/components/Sep6Send.tsx

```diff
@@ -0,0 +1,183 @@
+import { useRedux } from "hooks/useRedux";
+import { ActionStatus } from "types/types.d";
+
+export const Sep6Send = () => {
+  const { sep6DepositAsset } = useRedux("sep6DepositAsset");
+  const { depositResponse } = sep6DepositAsset;
+
+  const [formData, setFormData] = useState<any>({
+    depositType: {},
+    fields: {},
+  });
+  const dispatch = useDispatch();
+
+  const depositTypeChoices = useMemo(
+    () => sep6DepositAsset.data.depositTypes?.type?.choices || [],
+    [sep6DepositAsset],
+  );
+
+  useEffect(() => {
+    if (sep6DepositAsset.status === ActionStatus.NEEDS_INPUT) {
+      setFormData({
+        depositType: {
+          type: depositTypeChoices[0],
+        },
+        fields: {},
+      });
+    }
+  }, [sep6DepositAsset.status, depositTypeChoices, dispatch]);
+
+  useEffect(() => {
+    if (sep6DepositAsset.status === ActionStatus.CAN_PROCEED) {
+      dispatch(sep6DepositAction());
+    }
+  }, [sep6DepositAsset.status, dispatch]);
+
+  const resetLocalState = () => {
+    setFormData({});
+  };
+
+  const handleClose = () => {
+    dispatch(resetSep6DepositAction());
+    dispatch(resetActiveAssetAction());
+    resetLocalState();
+  };
+
+  const handleDepositTypeChange = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ) => {
+    const { id, value } = event.target;
+
+    const updatedState = {
+      ...formData,
+      depositType: {
+        ...formData.depositType,
+        [id]: value,
+      },
+    };
+
+    setFormData(updatedState);
+  };
+
+  const handleFieldChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { id, value } = event.target;
+
+    const updatedState = {
+      ...formData,
+      fields: {
+        ...formData.fields,
+        [id]: value,
+      },
+    };
+
+    setFormData(updatedState);
+  };
+
+  const handleSubmit = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  ) => {
+    event.preventDefault();
+    dispatch(submitSep6DepositFields({ ...formData }));
+  };
+
+  if (sep6DepositAsset.status === ActionStatus.NEEDS_INPUT) {
+    return (
+      <Modal visible={true} onClose={handleClose}>
+        <Heading2
+          className="ModalHeading"
+          tooltipText={
+            <>
+              These are the fields the receiving anchor requires. The sending
+              client obtains them from the /customer endpoint.{" "}
+              <TextLink
+                href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get"
+                isExternal
+              >
+                Learn more
+              </TextLink>
+            </>
+          }
+        >
+          SEP-6 Required Info
+        </Heading2>
+        <div className="ModalBody">
+          {Object.entries(sep6DepositAsset.data.depositTypes || {}).map(
+            ([id, input]) => (
+              <>
+                <label>{input.description}</label>
+                <Select id={id} key={id} onChange={handleDepositTypeChange}>
```

Comment:
`<Select>` has a `label` prop, can't we use it here?

Reply:
Ah, missed that! Yes, I should use the label prop